lib/active_merchant/billing/gateways/kushki.rb

```diff
@@ -204,6 +206,33 @@ def add_deferred(post, options)
         }
       end
 
+      def add_three_d_secure(post, payment_method, options)
+        three_d_secure = options[:three_d_secure]
+        return unless three_d_secure
```

Comment:
💬 Comment:

For extra security and standards ensure the guard with `three_d_secure.present?`

Reply:
Thanks a lot for the suggestion.